go.mod

```diff
@@ -12,11 +12,10 @@ require (
 	github.com/spf13/afero v1.6.0
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
-	github.com/topfreegames/maestro v1.0.1-0.20220107213135-829d51ff5769
+	github.com/topfreegames/maestro v1.0.1-0.20220203222207-0bf38a104155
```

Comment:
@lyracampos is this version already pointing to `next` branch?

Reply:
@GuilhermBrSp It's not. I changed it to the last commit of the next branch with the necessary changes.